README.md

```diff
@@ -121,6 +121,18 @@ ReactDOM.createRoot(document.getElementById("root") as HTMLElement)
 
 ```
 
+### Custom Language Support
+
+By default `prism-react-renderer` only includes a [base set of languages](https://github.com/FormidableLabs/prism-react-renderer/blob/c914fdea48625ba59c8022174bb3df1ed802ce4d/packages/generate-prism-languages/index.ts#L9-L23) that Prism supports. **Depending on your app's build system you may need to `await` the `import` or use `require` to ensure `window.Prism` exists before importing the custom languages.** You can add support for more by including their definitions from the main `prismjs` package:
+
+```js
+import { Highlight, Prism } from "prism-react-renderer";
+
+typeof window !== "undefined" && (window.Prism = Prism)

```

Comment:
The old recommendation was this:

```ts
(typeof global !== "undefined" ? global : window).Prism = Prism;
```

I'm assuming so that this would work in Node environments. Think we should keep that here so that this recommendation works in Node as well, for SSR purposes?

Reply:
@gksander It does, but it actually blows up in our Vite demo as `global` doesn't exist. I'd assume people are gonna have to tailor it a bit to make it work for their app. `global` vs `window`. Do you think we should mention it might need to live on global too?

Reply:
What kind of error are you getting in vite? The global one worked for me when I just tried it, but I might be missing something.

Reply:
`global` is `undefined` and does not exist, which I thought was a strange error to get on `typeof global`